starlite/cache/redis_cache_backend.py

```diff
@@ -0,0 +1,87 @@
+from typing import Any, Optional
+
+from pydantic import BaseModel
+
+from starlite import MissingDependencyException
+
+try:
+    from redis.asyncio import Redis
+    from redis.asyncio.connection import ConnectionPool
+except ImportError as e:
+    raise MissingDependencyException(
+        "To use starlite.redis_cache_backend, install starlite with 'redis_cache_backend' extra, e.g. `pip install starlite[redis_cache_backend]`"
+    ) from e
+
+
+from starlite.cache.base import CacheBackendProtocol
+
+
+class RedisCacheBackendConfig(BaseModel):
+    url: str
+    db: Optional[int] = None
```

Comment:
please add attribute docstrings and add the class to the doc references.

Reply:
Done!